script/Arcueid/Day 9/09_00_ARC09_4.txt

```diff
@@ -71,7 +72,7 @@ I don't remember much about it at the very least.
 [sha:0607cfa8f1d558e981c5e807d1037385baa8a4a8]{
 -- Page 127, Offset 21632.
 -- 森の外からでは決して見えない死角で、屋敷にいる時は……翡翠に付いてこなければ一生気づかなかったぐらいの、木々に囲まれた小さな広場。
-In a blind spot that could never been from outside the forest, there is a small field surrounded by trees... If I hadn't followed Hisui, I may never have found this in my entire life.
+In a blind spot that could never be seen from outside the forest, there's a small field surrounded by trees... If I hadn't followed Hisui, I may have never found this place.
```

Comment:
In a spot invisible to the outside world, there's a small clearing in the forest... If I hadn't followed Hisui, I may have found this place hidden by the trees.